server.py

```diff
@@ -76,10 +75,10 @@ def store_data(self, key, value):
     def temp_all_contents(self):
         new_cl = self.db["character_sheets"]
         try:
-            results = new_cl.find()
-            print(results)
-            print(type(results))
-            return str(results)
+            results = []
+            for i in new_cl.find():
+                results.append(i)
+            return results
```

Comment:
I'm surprised that both of these work, you tested the new one?

Reply:
the deleted one didn't work (maybe after these changes). latter works, frontend logs the object of all the fields in the table